bundles/core/src/test/java/com/adobe/cq/wcm/core/components/internal/servlets/DMAssetPostProcessorTest.java

```diff
@@ -361,7 +361,29 @@ private void assertNewModification(String component, List<Modification> modifica
             assertEquals(actualModification.getType(), expectedModification.type);
         }
     }
+//
+    /*
+     The case when DM asset drug-n-drop on empty page. This scenario creates Image component with assigned asset.
+     Post processor should write image server url property
+     */
+    @ParameterizedTest
+    @EnumSource(
+        value = ModificationType.class,
+        names = {"CREATE", "MODIFY"}
+    )
+    public void drugNdropDMimageToEmptyPage(ModificationType modificationType) throws Exception {
+        String expectingComponent = "/content/test/jcr:content/root/container/image40";
+        prepareResource(expectingComponent, null);
```

Comment:
@oborysyu I think simulated request is missing additional properties which will tell sling to create nested CoreImage